prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/GU/GU-receiver-transform.yml

```diff
@@ -0,0 +1,46 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+constants:
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+  ORC: '/PATIENT_RESULT/ORDER_OBSERVATION/ORC'
+  OBR: '/PATIENT_RESULT/ORDER_OBSERVATION/OBR'
+  SPM: '/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM'
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: gu-name-of-the-sender-application-Namespace-Id
+    condition: 'true'
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ 'MSH-3-1' ]
+
+  - name: gu-sender-application-Universal-Id
+    condition: 'true'
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ 'MSH-3-2' ]
+
+  - name: gu-sender-application-Universal-Id-Type
+    condition: 'true'
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-3-3' ]
+
+  - name: gu-name-of-the-receiving-application
+    condition: 'true'
+    value: [ '"GUDOH"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: gu-receiving-facility-namespace-ID
+    condition: 'true'
+    value: [ '"FDOH"' ]
```

Comment:
I think this needs to be GUDOH, that is what was in their replaceAwithB setting
"MSH-6": [
      {
        "*": "GUDOH"
      }
    ]

Reply:
Good catched.